lib/services/ios-log-filter.ts

```diff
@@ -41,14 +41,16 @@ export class IOSLogFilter extends iOSLogFilterBase.IOSLogFilter implements Mobil
 				// This code removes unnecessary information from log messages. The output looks like:
 				// CONSOLE LOG file:///location:row:column: <actual message goes here>
 				if (pid) {
-					const searchString = "[" + pid + "]: ";
-					const pidIndex = line.indexOf(searchString);
-					if (pidIndex > 0) {
-						line = line.substring(pidIndex + searchString.length, line.length);
+					const pidIndex = line.indexOf(`[${pid}]`);
+					const ios11PidIndex = line.indexOf(` ${pid} `);
+					if (pidIndex > 0 || ios11PidIndex > 0) {
+						const pidRegex = new RegExp(`.*[\\[\\s]{1}${pid}[\\s\\]]{1}.*?:(?:.*?\\(NativeScript\\) )*`);
```

Comment:
Why do we need the part after `:` in this `RegExp` - I'm talking about the ignored capture group `` (?:.*?\\(NativeScript\\) )* ``

If one should wish to `console.log` something like `Loving (NativeScript) right now` in their application the log would appear as
`"Aug 22 10:59:20 MCSOFAPPBLD TestApp[52946]: CONSOLE LOG file:///app/home/home-view-model.js:6:20: Loving (NativeScript) right now"`

and the `RegExp` would strip it to `right now`

Reply:
You are right will fix the regex.